server/routes/admin.js

```diff
@@ -4,13 +4,11 @@ import AuthorController from '../controllers/AuthorController';
 
 const admin = express.Router();
 const BASE_URL = '/admin';
-const { AuthorValidator } = middlewares;
-// const { verifyToken, authorizeUser, AuthorValidator } = middlewares;
+const { verifyToken, authorizeUser, AuthorValidator } = middlewares;
 const { addAuthor } = AuthorController;
 const { authorValidation } = AuthorValidator;
-// const authorizeAdmin = authorizeUser(['admin']);
+const authorizeAdmin = authorizeUser(['admin']);
 
-admin.post(`${BASE_URL}/author`, authorValidation(), addAuthor);
-// admin.post(`${BASE_URL}/author`, authorValidation(), verifyToken, authorizeAdmin, addAuthor);
+admin.post(`${BASE_URL}/author`, authorValidation(), verifyToken, authorizeAdmin, addAuthor);
```

Comment:
verifyToken should come before any other middleware on this route, then authorizeAdmin.

Reply:
> verifyToken should come before any other middleware on this route, then authorizeAdmin.

Thought so too, but didn't want to alter what was written before. I've corrected it now. Please re-review.